x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -45,10 +45,16 @@ export class HeadlessChromiumDriver {
   private readonly inspect: boolean;
   private readonly networkPolicy: NetworkPolicy;
 
+  private hasCdpListener: boolean;
+  private hasPupListener: boolean;
```

Comment:
Sorry, I don't know what `cdp` / `pup` means. Could you add quick comments to help challenged readers like me? 😄

Reply:
Phased these out for a better named thing